src/Renderer/renderer.js

```diff
@@ -25,6 +25,13 @@ class gltfRenderer
         this.webGl = new gltfWebGl();
         this.webGl.context = context;
 
+        // create render target for non transmission materials
+        this.opaqueRenderTexture = 0;
+        this.opaqueFramebuffer = 0;
+        this.opaqueDepthTexture = 0;
+        this.opaqueFramebufferWidth = 1024;
+        this.opaqueFramebufferHeight = 1024;
```

Comment:
maybe we should just call them after transmission, opaque could be confused with the traditional opaque / alpha blending approach

Reply:
This framebuffer for now contains all opaque elements. "Opaque" in this context excludes transmissive and alpha-blended objects. So I think, we can keep this name